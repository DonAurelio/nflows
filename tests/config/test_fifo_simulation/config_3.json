{
    "dag_file": "./tests/workflows/test_fifo_simulation/config_3.dot",
    "scheduler_type": "fifo",
    "mapper_type": "simulation",

    "core_avail_mask": "0x0F",
    "flops_per_cycle": 1000000,
    "clock_frequency_type": "static",
    "clock_frequency_hz": 1,

    "distance_matrices": {
        "latency": "./tests/system/test_fifo_simulation/3_lat.txt",
        "bandwidth": "./tests/system/test_fifo_simulation/3_bw.txt"
    },

    "log_base_name": "./tests/output/test_fifo_simulation/config_3",
    "log_date_format": ""
}
