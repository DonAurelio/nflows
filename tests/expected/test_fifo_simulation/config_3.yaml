runtime:
  core_availability:
    0: {avail_until: 90}
    1: {avail_until: 60}
    2: {avail_until: 70}
    3: {avail_until: 100}

trace:
  exec_name_total_offsets:
    Task_3: {start: 70, end: 90, payload: 10}
    Task_4: {start: 70, end: 100, payload: 10}
    Task_2: {start: 30, end: 70, payload: 10}
    Task_5: {start: 30, end: 60, payload: 10}
    Task_1: {start: 0, end: 30, payload: 10}
